### DISCUSSION TITLE
How i can translate the docs to Portuguese?

### DISCUSSION THREAD

**Discussion by GiovaneVerbinnen** [vuepress/core#315] [Ideas]
I see the only have English and Chinese, i wanna make a pt-BR version, it's allowed?

**Comment by meteorlxy** [comment]
You can consider maintaining the translation in a seperate repo, because it's unrealistic to keep all translations in a single official repo. 

In fact, the i18n mechanism of VuePress is not suitable for large scale documentation.

**Reply by GiovaneVerbinnen** [reply]
I understand, in fact i will make some articles or fork the project and translate for some people who don't  can learn, grateful for the feedback.
